### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Geartrixy**

### CODE REVIEW

**Comment by Geartrixy** [comment]
@derBroBro please check and merge.  :-)

File: security_groups.tf [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Security group "group" for loadbalancer with no rules. To be used in other security group rules to specify the loadblaancer as the source.
+resource "aws_security_group" "lb_group" {
+  name        = "Group-ALB-${var.lb_name}"
+  description = "Attach security group with loadbalancer name to loadbalancer with no rules"
+  vpc_id      = "${var.vpc_id}"
+}
+# Security group which allows all outbound traffic
+resource "aws_security_group" "rule_all_out_2" {
+  name        = "Rule-all-out-all-2"
```
Comment (derBroBro) [inline security_groups.tf]: Name needs to be changed.

File: variables_load_balancer.tf [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+## Load balancer security groups
+# Load balancer additional security groups
+variable "lb_security_group_ids" {
+  type        = "list"
+  description = "Additional Security Group ID(s) which should be attached to the loadbalancer"
+  default = []
+}
+# Loadbalancer source traffic name (for security group allowing listener traffic in)
+variable "lb_source_traffic_name" {
+  #HTTP security group name: "Rule-allow-${var.lb_source_traffic_name}-in-HTTP"
```
Comment (derBroBro) [inline variables_load_balancer.tf]: Why the comment?

File: variables_load_balancer.tf [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+  #HTTPS security group name: "Rule-allow-${var.lb_source_traffic_name}-in-HTTPS"
```
Comment (derBroBro) [inline variables_load_balancer.tf]: Why the comment?

**Review by derBroBro — CHANGES_REQUESTED** [review]
Wow, great module.
The are no blocking changes, just small optimisations.
Please also make a "terraform fmt" to beautify all files. :-)

**Comment by Geartrixy** [comment]
Changes made to all out group name.
Amended description for variable "lb_source_traffic_name".
Terraform fmt.

**Review by derBroBro — APPROVED** [review]

**Pull request merged**